frontend/sw360-portlet/src/main/webapp/html/components/view.jsp

```diff
@@ -218,12 +228,23 @@
     }
 
     function exportExcel(){
+        $('#keywordsearchinput').val("");
```

Comment:
I tried it again: if you apply a (real) filter and thereafter the KEYWORD SEARCH, then you are allowed to click export excel. The KEYWORD_SEARCH input stays visible in the input field, but in the exported excel, this is not respected. That would be inconsistent.